packages/core/vite.config.ts

```diff
@@ -0,0 +1,11 @@
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  plugins: [sveltekit()],
+  test: {
+    include: ['src/**/*.{test,spec}.{js,ts}'],
+    globals: true,
```

Comment:
Rather than setting `globals: true`, we can set up a `vitest.setup.ts` file to install `@testing-library/jest-dom` and configure the testing-library reset. I softly prefer it to globals mode which feels to me like a silly holdover from pre-module times